astroquery/nasa_exoplanet_archive/__init__.py

```diff
@@ -1,10 +1,11 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
-
-
 import warnings
 
 warnings.warn("the ``nasa_exoplanet_archive`` module has been moved to "
               "astroquery.ipac.nexsci.nasa_exoplanet_archive, "
               "please update your imports.", DeprecationWarning, stacklevel=2)
 
-from astroquery.ipac.nexsci.nasa_exoplanet_archive import *
+from astroquery.ipac.nexsci.nasa_exoplanet_archive import NasaExoplanetArchive, NasaExoplanetArchiveClass, Conf
+
+
+__all__ = ["NasaExoplanetArchive", "NasaExoplanetArchiveClass", "Conf"]
```

Comment:
There's at least one case of `__all__` missing `conf`, there might be more.
```suggestion
__all__ = ["NasaExoplanetArchive", "NasaExoplanetArchiveClass", "Conf", "conf"]
```

Reply:
this module is deprecated, and will be removed, no need to fix this.